hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/gas/DispatchType.java

```diff
@@ -61,7 +61,8 @@ public enum DispatchType {
     TOKEN_INFO(HederaFunctionality.TOKEN_GET_INFO, DEFAULT),
     UPDATE_TOKEN_CUSTOM_FEES(HederaFunctionality.TOKEN_FEE_SCHEDULE_UPDATE, DEFAULT),
     TOKEN_AIRDROP(HederaFunctionality.TOKEN_AIRDROP, DEFAULT),
-    TOKEN_CLAIM_AIRDROP(HederaFunctionality.TOKEN_CLAIM_AIRDROP, DEFAULT);
+    TOKEN_CLAIM_AIRDROP(HederaFunctionality.TOKEN_CLAIM_AIRDROP, DEFAULT),
+    TOKEN_REJECT(HederaFunctionality.TOKEN_REJECT, TOKEN_FUNGIBLE_COMMON);
```

Comment:
(per discussion elsewhere:). Probably needs to have two entries fungible + NF (like MINT/BURN/WIPE above)

Reply:
Added gas calculation based on selector.